npm run build fails

```
$ sudo npm run build                                                                                                                                                              shark@rawsec

> flood@1.0.0 build /srv/flood
> node client/scripts/build.js

Creating an optimized production build...
Failed to compile.

./client/src/sass/style.scss
Module build failed: Error: ENOENT: no such file or directory, scandir '/srv/flood/node_modules/node-sass/vendor'

./client/src/sass/style.scss
Module build failed: ModuleBuildError: Module build failed: Error: ENOENT: no such file or directory, scandir '/srv/flood/node_modules/node-sass/vendor'

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flood@1.0.0 build: `node client/scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the flood@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-11-11T17_49_10_048Z-debug.log
```

`/root/.npm/_logs/2017-11-11T17_49_10_048Z-debug.log`:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'build' ]
2 info using npm@5.5.1
3 info using node@v8.8.1
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle flood@1.0.0~prebuild: flood@1.0.0
6 info lifecycle flood@1.0.0~build: flood@1.0.0
7 verbose lifecycle flood@1.0.0~build: unsafe-perm in lifecycle true
8 verbose lifecycle flood@1.0.0~build: PATH: /usr/lib/node_modules/npm/bin/node-gyp-bin:/srv/flood/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
9 verbose lifecycle flood@1.0.0~build: CWD: /srv/flood
10 silly lifecycle flood@1.0.0~build: Args: [ '-c', 'node client/scripts/build.js' ]
11 silly lifecycle flood@1.0.0~build: Returned: code: 1  signal: null
12 info lifecycle flood@1.0.0~build: Failed to exec build script
13 verbose stack Error: flood@1.0.0 build: `node client/scripts/build.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:280:16)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at EventEmitter.emit (events.js:213:7)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at ChildProcess.emit (events.js:213:7)
13 verbose stack     at maybeClose (internal/child_process.js:927:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
14 verbose pkgid flood@1.0.0
15 verbose cwd /srv/flood
16 verbose Linux 3.14.32-xxxx-grs-ipv6-64
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "build"
18 verbose node v8.8.1
19 verbose npm  v5.5.1
20 error code ELIFECYCLE
21 error errno 1
22 error flood@1.0.0 build: `node client/scripts/build.js`
22 error Exit status 1
23 error Failed at the flood@1.0.0 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

```
$ node --version
v8.8.1
$ npm --version
5.5.1
$ node-gyp --version
v3.6.2
```

Another error that can happen when having an outdated node-sass:

```
npm run build

> flood@1.0.0 build /srv/flood
> node client/scripts/build.js

Creating an optimized production build...
Failed to compile.

./client/src/sass/style.scss
Module build failed: Error: Missing binding /srv/flood/node_modules/node-sass/vendor/linux-x64-59/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 9.x

Found bindings for the following environments:
  - Linux 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass --force` to build the binding for your current environment.

./client/src/sass/style.scss
Module build failed: ModuleBuildError: Module build failed: Error: Missing binding /srv/flood/node_modules/node-sass/vendor/linux-x64-59/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 9.x

Found bindings for the following environments:
  - Linux 64-bit with Node.js 8.x

This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass --force` to build the binding for your current environment.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! flood@1.0.0 build: `node client/scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the flood@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/flood/.npm/_logs/2017-11-26T23_30_01_613Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -66,7 +66,7 @@
     "multer": "^1.4.1",
     "mv": "^2.1.1",
     "nedb": "^1.8.0",
-    "node-sass": "^4.10.0",
+    "node-sass": "^4.11.0",
     "object-assign": "4.1.1",
     "ospath": "^1.2.2",
     "passport": "^0.4.0",

